internal/cmd/server/main.go

```diff
@@ -0,0 +1,47 @@
+// Package main provides the main entry point for the application.
+package main
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/ezex-io/ezex-users/internal/config"
+	"github.com/ezex-io/ezex-users/internal/core/server"
+	"github.com/ezex-io/ezex-users/internal/core/service"
+	"github.com/ezex-io/ezex-users/internal/infra/repository"
+	"github.com/ezex-io/gopkg/logger"
+)
+
+var log = logger.NewSlog(nil)
+
+func main() {
+	cfg, err := config.Load()
+	if err != nil {
+		log.Fatal("Failed to load config", "error", err)
+	}
+
+	repo := repository.NewSecurityImageRepository()
```

Comment:
I don't think this is correct. This service will cover more items, such as the user's profile, settings, etc.
The security image is just one item.
Do we need a new instance for each task? I don't think so.